presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -395,10 +399,10 @@ public void testParquetThrift()
     public void testDwrf()
             throws Exception
     {
-        List<TestColumn> testColumns = ImmutableList.copyOf(filter(TEST_COLUMNS, testColumn -> {
-            ObjectInspector objectInspector = testColumn.getObjectInspector();
-            return !hasType(objectInspector, PrimitiveCategory.DATE);
-        }));
+        List<TestColumn> testColumns = TEST_COLUMNS.stream()
+                .filter(testColumn -> all(ImmutableList.of(PrimitiveCategory.DATE, PrimitiveCategory.VARCHAR),
+                        primitiveCategory -> !hasType(testColumn.getObjectInspector(), primitiveCategory)))
```

Comment:
`hasType` takes varargs, so this can just be `!hasType(objectInspector, PrimitiveCategory.DATE, PrimitiveCategory.VARCHAR)`